data/docs/migration/migrate-from-grafana.mdx

```diff
@@ -0,0 +1,271 @@
+
+### Requesting Custom Dashboards
+
+You can also request SigNoz to help you with creating custom dashboards. To request a custom dashboard, please create a [GitHub issue](https://github.com/SigNoz/signoz/issues/new?assignees=&labels=dashboard-template&template=request_dashboard.md&title=%5BDashboard+Request%5D+) providing details about the dashboard you want to create.
+
+### Dashboard Features
+
+SigNoz supports essential dashboard features including:
+
+- **[Variables](/docs/userguide/manage-variables/)**: Create template variables for dynamic dashboards
+- **[Panel Types](/docs/dashboards/panel-types/)**: Various visualization options (time series, bar charts, tables, etc.)
+- **[Query Builder](/docs/product-features/query-builder/)**: Construct queries visually or using PromQL
+
+### Panel Types
+
+SigNoz supports most of the common panel types supported by Grafana:
+
+| Panel Type      | Grafana | SigNoz | Notes                                                                                           |
+|----------------|---------|---------|------------------------------------------------------------------------------------------------|
+| Time Series    | ✓       | ✓       | Both platforms support time series visualization with thresholds and unit configuration         |
+| Bar Chart      | ✓       | ✓       | Both support categorical data visualization                                                     |
+| Pie Chart      | ✓       | ✓       | Both support proportional data visualization                                                    |
+| Table          | ✓       | ✓       | Both support tabular data display                                                              |
+| Histogram      | ✓       | ✓       | Both support distribution visualization                                                         |
+| Value/Stat     | ✓       | ✓       | Single value display with thresholds                                                           |
+| Gauge          | ✓       | -       | Similar functionality available via [Value panels](/docs/dashboards/panel-types/value/) with thresholds                                |
+| Node Graph     | ✓       | -       | [Service Map](/docs/userguide/service-map/) provides similar functionality for service dependencies                             |
+| Logs           | ✓       | ✓       | Dedicated [List panel type](/docs/dashboards/panel-types/list/) for logs visualization                                               |
+| Traces         | ✓       | ✓       | Advanced trace visualization through [APM features](/docs/userguide/traces/)                                              |
+| Flame Graph    | ✓       | ✓       | Both support flame graphs for performance profiling                                            |
```

Comment:
SigNoz doesn't have flamegraph panel type in dashboards

Reply:
Added clarification that it's available as part of APM.